app/src/main/java/com/BetterTogether/app/Dialogs/PopupView.java

```diff
@@ -5,6 +5,8 @@
 import android.view.LayoutInflater;
 import com.BetterTogether.app.Fragments.UserListFragment;
 
+import com.BetterTogether.app.R;
```

Comment:
R?

Reply:
Android things, it's for "Resources". Need it for adding a colour theme to popupView  (R.style.MyTheme).

Reply:
Is this an android naming conversion thing?

Reply:
It's not mine. It comes from Android, so I can't do anything about the name.